github/Organization.py

```diff
@@ -599,6 +600,7 @@ def create_team(
         :param permission: string
         :param privacy: string
         :param description: string
+        :param parent_team_id: string
```

Comment:
This states parent_team_id is string, but the typing hint says int.

Reply:
Updated with https://github.com/PyGithub/PyGithub/pull/1669/commits/2c272f8b58e14cfbde843f07967a7003a62dae78